docs-src/docs/usingwidgets.md

```diff
@@ -0,0 +1,163 @@
+# Using Widgets
+
+[Widgets](widgetoverview.md) are how you create your GUI. They are the things which appear on the GUI, everything from the app itself to text boxes, buttons and pictures.
+
+At the start of every guizero program, choose the widgets you need from the guizero library and import them e.g.
+
+```python
+from guizero import App, Text, PushButton, Slider
+```
+
+You only need to import each widget once, and then you can use it in your program as many times as you like.
+
+### The App widget
+
+Every guizero app will need an [App](app.md) widget. This is the main window.
+
+At the end of every guizero program you must tell the program to `display` the app you have just created.
+
+```python
+from guizero import App
+
+app = App(title="My guizero app")
+
+app.display()
+```
+
+### Adding widgets
+
+All the text boxes, buttons, sliders and even plain old pieces of text which appear on your GUI are widgets.
+
+**All widgets** go between the line of code to create the `App` and the `app.display()` line.
+
+```python
+from guizero import App, Text, TextBox
+
+app = App(title="Whats your name?")
+
+# create widgets here
+message = Text(app, text="Enter your name")
+name = TextBox(app)
+
+app.display()
+```
+
+Let’s look at the [Text](text.md) widget code in a bit more detail:
+
+```python
+message = Text(app, text="Enter your name"))
+```
+
+- `message =` - The `Text` object has a name, just like any variable
+- `Text` - an *object* which creates a piece of text on the screen
+- `app` – This tells the `Text` where it will live. Most of the time your widgets will live directly inside the app.
+- `text="Enter your name"` - The text to display
+
+![an app showing the text 'Enter your name` with a text box below where a name can be entered](images/whats-your-name.png)
+
+Have a look at [Widget overview](widgetoverview.md) to see all the widgets, find out more about them and how they are used.
+
+### Using events
+
+An event is how you trigger code as a result of users interacting with your app. The event could be a user clicking on a button or entering text into a TextBox. When an event occurs you can run a piece of code. 
```

Comment:
_Examples of events could be_ a user clicking on a button....

(i.e. those aren't the only possible events)